src/TestProblem/Hydro/BlastWave/Init_TestProb_Hydro_BlastWave.cpp

```diff
@@ -223,6 +223,17 @@ void SetGridIC( real fluid[], const double x, const double y, const double z, co
    MomY = 0.0;
    MomZ = 0.0;
    Pres = ( r <= Blast_Radius ) ? Blast_Pres_Exp : Blast_Pres_Bg;
+#  ifdef SRHD
+   real Prim[NCOMP_TOTAL];
+   Prim[0] = Dens;
+   Prim[1] = MomX;
+   Prim[2] = MomY;
+   Prim[3] = MomZ;
```

Comment:
It is a little bit confusing to assign a primitive variable to a conserved variable. Can you change it to `Prim[*] = 0.0;`?